cpu/nrf51/periph/pwm.c

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     cpu_nrf51
+ * @ingroup     drivers_periph_pwm
+ * @{
+ *
+ * @file
+ * @brief       Low-level PWM driver implementation
+*
+ * @note        This driver initializes a duty cycle of '1'
+ *
+ * @author      Semjon Kerner <semjon.kerner@fu-berlin.de>
+ * @}
+ */
+
+#include <stdint.h>
+#include <string.h>
+#include <math.h>
+
+#include "periph/gpio.h"
+#include "periph/pwm.h"
+#include "vectors_cortexm.h"
+
+#define ENABLE_DEBUG                (1)
+#include "debug.h"
+#define DEBUG_ASSERT_VERBOSE        (1U)
+
+#define NRF51_PWM_PRESCALER_MIN     (0U)
```

Comment:
as these are local defines, I usually don't prefix them with the full module names. IMHO this makes the code better to understand, as it is clear which defines are local and which ones come from a global context. But this might be subjective?!